src/middleware.ts

```diff
@@ -0,0 +1,42 @@
+import { NextRequest, NextResponse } from "next/server";
+
+// 1. Specify protected and public routes
+const protectedRoutes = [
+  "/dashboard",
+  "/coaching-sessions",
```

Comment:
@zgavin1 I'm still able to visit /coaching-sessions/[id] when logged out. Are you seeing this too?

Reply:
yep good catch